Gemfile

```diff
@@ -32,6 +32,8 @@ gem 'jbuilder', '~> 2.5'
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+gem 'filterrific'
```

Comment:
It is completely fine using a gem for this purpose. But for the sake of learning in demo projects, we usually encourage you to build most of the stuff on your own. Please remove this gem and build this search/filter feature on your own so that you get play more with it.